lib/CSSStyleDeclaration-impl.js

```diff
@@ -9,23 +9,24 @@ var allExtraProperties = require('./allExtraProperties');
 var implementedProperties = require('./implementedProperties');
 var { dashedToCamelCase } = require('./parsers');
 var getBasicPropertyDescriptor = require('./utils/getBasicPropertyDescriptor');
+const idlUtils = require('./utils.js');
 
-/**
- * @constructor
- * @see http://www.w3.org/TR/DOM-Level-2-Style/css.html#CSS-CSSStyleDeclaration
- */
-var CSSStyleDeclaration = function CSSStyleDeclaration(onChangeCallback) {
-  this._values = {};
-  this._importants = {};
-  this._length = 0;
-  this._onChange =
-    onChangeCallback ||
-    function() {
-      return;
-    };
-};
-CSSStyleDeclaration.prototype = {
-  constructor: CSSStyleDeclaration,
+class CSSStyleDeclaration {
+  /**
+   * @constructor
+   * @see http://www.w3.org/TR/DOM-Level-2-Style/css.html#CSS-CSSStyleDeclaration
```

Comment:
Old! Looks like the rest of the file all have this though, so let's defer this to another PR.